app/population/controllers/edit/details.js

```diff
@@ -0,0 +1,4 @@
+const EditPopulationBaseController = require('./base')
+
+class DetailsController extends EditPopulationBaseController {}
```

Comment:
I left a comment on the steps object around this, but if you move the initial values method and the save method to this controller you can reduce the amount of inheritance and files needed.

Am I right in assuming this is only ever going to be a one-step form?

Reply:
Refactored to only use one controller.